Violin.m

```diff
@@ -504,9 +527,9 @@
                 
                 
         function set.ViolinAlpha(obj, alpha)
-            obj.ViolinPlotQ.FaceAlpha = .8;
+            obj.ViolinPlotQ.FaceAlpha = .65; % reverse to .8?
             obj.ViolinPlot.FaceAlpha = alpha{1};
-            obj.ScatterPlot.MarkerFaceAlpha = alpha{1};
+            obj.ScatterPlot.MarkerFaceAlpha = 1;% reverse to alpha{1}? This increases visibility

```

Comment:
I think comments such as these should be deleted from the repo before merging.

And personally, I'd be very weary about changing default values, even for good reasons. Existing users might rely on existing functionality. At the very least we should provide a clear changelog whenever a new version is released, so users can find the cause of any regression quickly.